ops/ansible/aws/deploy-mesh/tasks/main.yml

```diff
@@ -88,7 +88,7 @@
   become: yes
   shell: |
     echo "Running command: nexd --username {{ nexodus_auth_uid }} --password {{ nexodus_auth_password }} {{ nexodus_url }}" > nexodus-logs.txt
-    nexd \
+    NEXD_LOGLEVEL=debug nexd \
```

Comment:
You want to keep the ansible related debug changes ?

Reply:
yessir